cmd/monitor/verify.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2021 The Project Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
```

Comment:
Probably want a blank line and a comment about what this tool does and any assumptions about the env (e.g. `git` and `tamago` installed, `TAMAGO` env set, etc.?)

Reply:
This isn't the main file, but I've added more commentary to the main file to this effect. One thing to be considered is whether we want this as a separate tool altogether though? My view is that the monitor as it was was really the foundation for this type of full verification, but perhaps others will have other ideas.

Reply:
I am fine with having everything in the same tool.